manager/manager/templatetags/manager.py

```diff
@@ -232,3 +232,13 @@ def has_expired(errors) -> bool:
 
 
 register.filter("has_expired", has_expired)
+
+@register.simple_tag
+def url_replace(request, field, value):
+    dict_ = request.GET.copy()
```

Comment:
using the variable type as variable name is bad practice. Please use a name that's meaningful.